plugins/inventory/foreman.py

```diff
@@ -68,6 +68,10 @@
       host_filters:
         description: This can be used to restrict the list of returned host
         type: string
+      batch_size:
+        description: Determine the number of hosts that will be returned by the Foreman API per call
```

Comment:
```suggestion
        description: Number of hosts per batch that will be retrieved from the Foreman API per individual call
```
I think we should try to express this from the viewpoint of the ansible inventory.